templates/donors.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="donors">
+  <div class="donors_image">
+    {{{asset image}}}
+  </div>
+  <div class="donors_info">
+    <p class="donors_description">{{description}}</p>
+    <div class="donors_link_container">
+      <div class="donors_ctaPrimary_link" aria-label="Button for donations">
```

Comment:
Let's add a validation here for the `ctaPrimary` just to avoid any errors when the author does not provide one in the CMS, since this element is optional